### PR TITLE
[Validator] Document allowLowerCase allowSpaces options

### PR DESCRIPTION
**Pull request by alamirault**
see [https://github.com/symfony/symfony/pull/30272](https://github.com/symfony/symfony/pull/30272 )

### CODE REVIEW

File: reference/constraints/Bic.rst [hunk 1]
```diff
@@ -131,4 +133,20 @@ For example, if you want to compare the ``$bic`` property of some object
 with regard to the ``$iban`` property of the same object, use
 ``propertyPath="iban"`` in the comparison constraint of ``$bic``.
 
+allowLowerCase
+~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: false
+
+If this option is set to ``false`` and value contains a lower letter, a validation
```
Comment (OskarStark) [inline reference/constraints/Bic.rst]: Lowercase letter.... instead of lower letter... ?

Comment (OskarStark) [reply reference/constraints/Bic.rst]: The default is false and the explanation is using false too? Shouldn’t you explain what happens when setting this option to true?

I am on a phone right now... or is it the same for other options?

Comment (alamirault) [reply reference/constraints/Bic.rst]: I based myself on collection constraint documentation. allowMissingFields is to false by default and the explanation is using false too. But indeed I will append case when this options is to true

File: reference/constraints/Bic.rst [hunk 2]
```diff
@@ -131,4 +133,20 @@ For example, if you want to compare the ``$bic`` property of some object
 with regard to the ``$iban`` property of the same object, use
 ``propertyPath="iban"`` in the comparison constraint of ``$bic``.
 
+allowLowerCase
+~~~~~~~~~~~~~~~~
```
Comment (wouterj) [inline reference/constraints/Bic.rst]: The title underline should match the length of the title. Can you please remove characters to make sure they have the same length? (same applies to the title below)

**Review by wouterj — APPROVED** [review]

**Comment by OskarStark** [comment]
The related PR wasn't merged

**Comment by javiereguiluz** [comment]
Closing because the related PR was closed. This is always a bummer ... but there will be more opportunities to contribute. Thanks!

**Comment by alamirault** [comment]
No problem, I understand !

**Pull request closed without merging**